src/main/java/org/prebid/server/bidder/yearxero/YearxeroBidder.java

```diff
@@ -0,0 +1,84 @@
+package org.prebid.server.bidder.yearxero;
+
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import org.apache.commons.collections4.CollectionUtils;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.json.DecodeException;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.BidderUtil;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class YearxeroBidder implements Bidder<BidRequest> {
```

Comment:
as I can judge this bidder can be implemented as an alias of the generic bidder since the logic is straightforward

so please, check the `generic.yaml` and find there examples of the aliases

Reply:
Do you plan on using the adapter before adding the updates you mentioned?

Reply:
Yes, we would like to start as soon as possible and we have a list of improvements for our bidder in our backlog for implementing. We are going to work in parallel on both sides: on adapter and bidder.

Reply:
Alright, if you have further plans on updating the bidder, let it be a separate one